core/src/main/java/org/eclipse/hono/util/RegistrationConstants.java

```diff
@@ -42,6 +42,12 @@
      */
     public static final String FIELD_MAPPER = "mapper";
 
+    /**
+     * The name of the mapper used. This mapper should be configured for the adapter and can be referenced using
+     * this field.
```

Comment:
"command mapper" instead of "mapper"? To have that javadoc be different from the one for `FIELD_MAPPER`.